VMBackup/main/handle.py

```diff
@@ -348,8 +319,82 @@ def update():
 
 def enable():
+    global backup_logger,hutil,error_msg,para_parser
     hutil.do_parse_context('Enable')
-    start_daemon();
+    try:
+        finalpath=str(os.getcwd())
+        commandToExec ="chmod -R +x "+finalpath
+        subprocess.call(commandToExec,shell=True)
+    except Exception as e:
+        backup_logger.log('In enable permissions not changed', True)
+    try:
+        backup_logger.log('starting to enable', True)
+
+        # handle the restoring scenario.
+        mi = MachineIdentity()
+        stored_identity = mi.stored_identity()
+        if(stored_identity is None):
+            mi.save_identity()
+        else:
+            current_identity = mi.current_identity()
+            if(current_identity != stored_identity):
+                current_seq_no = -1
+                backup_logger.log("machine identity not same, set current_seq_no to " + str(current_seq_no) + " " + str(stored_identity) + " " + str(current_identity), True)
+                hutil.set_last_seq(current_seq_no)
+                mi.save_identity()
+
+        hutil.exit_if_same_seq()
+        hutil.save_seq()
+
+        """
+        protectedSettings is the privateConfig passed from Powershell.
+        WATCHOUT that, the _context_config are using the most freshest timestamp.
+        if the time sync is alive, this should be right.
+        """
+        protected_settings = hutil._context._config['runtimeSettings'][0]['handlerSettings'].get('protectedSettings')
+        public_settings = hutil._context._config['runtimeSettings'][0]['handlerSettings'].get('publicSettings')
+        para_parser = ParameterParser(protected_settings, public_settings)
+
+        if(para_parser.commandStartTimeUTCTicks is not None and para_parser.commandStartTimeUTCTicks != ""):
+            utcTicksLong = long(para_parser.commandStartTimeUTCTicks)
+            backup_logger.log('utcTicks in long format' + str(utcTicksLong), True)
+            commandStartTime = convert_time(utcTicksLong)
+            utcNow = datetime.datetime.utcnow()
+            backup_logger.log('command start time is ' + str(commandStartTime) + " and utcNow is " + str(utcNow))
+            timespan = utcNow - commandStartTime
+            THIRTY_MINUTES = 30 * 60 # in seconds
+            # handle the machine identity for the restoration scenario.
+            total_span_in_seconds = timedelta_total_seconds(timespan)
+            backup_logger.log('timespan is ' + str(timespan) + ' ' + str(total_span_in_seconds))
+            if(abs(total_span_in_seconds) > THIRTY_MINUTES):
+                error_msg = 'the call time stamp is out of date. so skip it.'
+                exit_with_commit_log(error_msg, para_parser)
+
+        if(para_parser.taskId is not None and para_parser.taskId != ""):
+            taskIdentity = TaskIdentity()
+            taskIdentity.save_identity(para_parser.taskId)
+        temp_status= 'transitioning'
+        temp_result=0
```

Comment:
As we have different temp_msg here now, lets assign temp_result=CommonVariables.success